apps/fluent-tester/src/FluentTester/TestComponents/Separator/consts.ts

```diff
@@ -1,2 +1,10 @@
 export const HOMEPAGE_SEPARATOR_BUTTON = 'Homepage_Separator_Button';
 export const SEPARATOR_TESTPAGE = 'Separator_TestPage';
+
+/* E2E Testing Separator 1 */
+export const SEPARATOR_TEST_COMPONENT = 'Separator_Test_Component'; // A component on each specific test page
+export const SEPARATOR_ACCESSIBILITY_LABEL = 'E2E testing Separator accessibility label';
```

Comment:
Another question about whether testing a11y label makes sense here. Separator is strictly a visual component right (as in not interactable)?

Reply:
Again, I agree with this. I'll remove it